docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
```

Comment:
In the first diagram you have a line between proxy on node and etcd, but that path no longer exists (API server is the only client of etcd now).